tests/possible_values.rs

```diff
@@ -196,10 +215,26 @@ fn possible_values_output() {
     ));
 }
 
+#[test]
+fn possible_values_hidden_output() {
```

Comment:
This is the same test as `hide_single_possible_val` in `test/help.rs`. You can remove it here

Reply:
kindof, one is for the error message, one is for the help message I think.